src/software/jetson_nano/thunderloop.cpp

```diff
@@ -405,3 +408,23 @@ double Thunderloop::getCpuTemperature()
         return 0.0;
     }
 }
+
+void Thunderloop::sendErrorCodes()
```

Comment:
nit: this isn't sending error codes to thunderscope, rather, it's setting/updating the error codes. I think we should rename the method. The comments here and in the `.h` file should be updated to reflect it as well.